src/main/java/liuni/resources/LiUniResource.java

```diff
@@ -67,10 +70,10 @@ public Response fetchTimeline() {
             responseBuilder.entity(new TwitterTimelineModel(timeline).getTimeline());
         }
         catch (Exception e) {
-            e.printStackTrace();
             ErrorModel error = new ErrorModel();
             responseBuilder.status(error.getErrorStatus());
             responseBuilder.entity(error.getGeneralError());
+            logger.error(e.getStackTrace().toString() + "\n\nProduced an error with a " + error.getErrorStatus() + " code.");
```

Comment:
there's a better way of just logging the entire exception object.  Apply to the others as well